README.md

```diff
@@ -2,12 +2,10 @@
 # Fixing Polygons in OSM
 
 There are many different ways (multi)polygons in OpenStreetMap can be mapped
-correctly and there are even more ways in which they can be mapped incorrectly.
-Of the over 220 million (multi)polygons in OSM more than 100,000 contain
-mapping errors of one kind or another and about 250,000 are tagged *old style*
-with tags on the outer ways instead of on the relation making multipolygon
-tagging and processing much more complicated and much more expensive than it
-needs to be.
+correctly and there are even more ways in which they can be mapped incorrectly, 
+using *old style* - with tags on the outer ways instead of on the relation 
+making multipolygon tagging and processing much more complicated and much more 
+expensive than it needs to be.
```

Comment:
@pnorman, my point was that exact numbers will be misleading in txt file, we should use (near) real-time stats if possible; I have no concerns about text, but outdated numbers are very misleading.